scripts/ci/gitlab/pipeline/publish.yml

```diff
@@ -38,17 +39,50 @@
     - echo "SUBSTRATE_IMAGE_TAG=${IMAGE_TAG}"   | tee -a ./artifacts/$PRODUCT/build.env
     - cat ./artifacts/$PRODUCT/build.env
 
+.build-push-docker-image:
+  extends:
+    - .publish-refs
+    - .build-push-docker-image-common
+  variables:
+    IMAGE_PATH:                    parity/$PRODUCT
+    COMMON_USER:                   $Docker_Hub_User_Parity
+    COMMON_PASS:                   $Docker_Hub_Pass_Parity
+
+
+# publish image to docker.io/paritypr, (e.g. for later use in zombienet testing)
+.build-push-image-temporary:
+  extends:
+    - .build-refs
+    - .build-push-docker-image-common
+  variables:
+    IMAGE_PATH:                    paritypr/$PRODUCT_ALIAS
+    COMMON_USER:                   $PARITYPR_USER
+    COMMON_PASS:                   $PARITYPR_PASS
+
 publish-docker-substrate:
-  stage:                           publish
   extends:                         .build-push-docker-image
   needs:
     - job:                         build-linux-substrate
       artifacts:                   true
   variables:
     PRODUCT:                       substrate
 
+publish-docker-substrate-temporary:
+  extends:                         .build-push-image-temporary
+  needs:
+    - job:                         build-linux-substrate
+      artifacts:                   true
+  variables:
+    PRODUCT:                       substrate
+    PRODUCT_ALIAS:                 substrate-debug
```

Comment:
We could use [`docker.io/paritypr/substrate`](https://hub.docker.com/r/paritypr/substrate)  (not `substrate-debug`) which seems to be not used for more then 1 year.

@alvicsam what do you think?